front/app/containers/Admin/sideBar/LanguageSelectorPopup.tsx

```diff
@@ -36,53 +41,50 @@ export const LanguageSelectorPopup = ({ setIsOpen, isOpen }: Props) => {
   const selectedLocale = getSelectedLocale(locale);
 
   return (
-    <Popup
-      trigger={
-        <Box display="flex" justifyContent="space-between" w="100%">
+    <>
+      <Box display="flex" justifyContent="space-between" w="100%">
```

Comment:
You can make the Box `relative`, position the Dropdown inside and orient the popup based on it. Also we need to make sure it's fine on mobile (the side bar gets smaller) and that it works well with the user menu change. Let's maybe merge my branch with the rest of the popups and double-check?